src/ontology/mondo-edit.obo

```diff
@@ -334863,15 +334863,17 @@ relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/
 
 [Term]
 id: MONDO:0014598
-name: developmental and epileptic encephalopathy, 31
-def: "Any early infantile epileptic encephalopathy in which the cause of the disease is a mutation in the DNM1 gene." [MONDO:patterns/disease_series_by_gene]
+name: developmental and epileptic encephalopathy, 31A
+def: "Autosomal dominant neurologic disorder presenting with global developmental delay apparent in early infancy." [https://orcid.org/0000-0002-0587-4693, MONDO:patterns/disease_series_by_gene, OMIM:616346]
```

Comment:
This new definition is too broad. It means that "Any neurological disorder that is autosomal dominant and present with a global development delay that is apparent in early infancy" is a "developmental and epileptic encephalopathy, 31A"

Reply:
@yshwetar In the ontology, I do not see that this term is an "early infantile epileptic encephalopathy". In fact, "early infantile epileptic encephalopathy" was obsoleted and replaced by "developmental and epileptic encephalopathy".
I suggest that you update the definition to say: "any developmental and epileptic encephalopathy in which (...)", where "developmental and epileptic encephalopathy" is the parent term of this term.

Reply:
Comments addressed.